language/sv.json

```
@@ -1,272 +1,523 @@
+    "energy_intensity_machinery_biofuel": "Energiintensitet maskineri (biobränsle)",
+    "energy_intensity_machinery_diesel": "Energiintensitet maskineri (diesel)",
+    "energy_intensity_machinery_electricity": "Energiintensitet maskineri (elektricitet)",
+    "energy_intensity_natural_gas_commercial": "Energiintensitet naturgas kommersiell",
+    "energy_intensity_natural_gas_domestic": "Energiintensitet naturgas hus",
+    "energy_intensity_new_commercial_buildings_standard": "Energiintensitet nya kommersiella byggnader (standard)",
+    "energy_intensity_new_domestic_buildings_standard": "Energiintensitet nya hus (standard)",
+    "energy_intensity_personal_bev": "Energiintensitet elbil",
+    "energy_intensity_petrol_ice": "Energiintensitet bensin ICE",
+    "energy_intensity_railway": "Energiintensitet tåg",
+    "energy_intensity_sea_transport": "Energiintensitet sjötransport",
+    "energy_intensity_walking_biking": "Energiintensitet cykling/gång",
+    "energy_intensity_zero_emission_commercial": "Energiintensitet 0-utsläpp kommersiell",
+    "energy_intensity_zero_emission_domestic": "Energiintensitet 0-utsläpp hus",
+    "engine_efficiency_improvement_personal_vehicle": "Motor effektivitetsförbättring (personliga fordon)",
+    "external_meetings": "Externa möten",
+    "heavy_truck_eco_driving_savings_factor_hgv": "Tunga lastbilar eco-körnings besparningsfaktor",
+    "heavy_truck_efficiency_improvement_factor_hgv": "Tunga lastbilar effektivitetsförbättring",
+    "light_truck_eco_driving_savings_factor_lgv": "Lätta lastbilar eco-körnings besparningsfaktor",
+    "light_truck_efficiency_improvement_factor_lgv": "Lätta lastbilar effektivitetsförbättring",
+    "load_factor_bus_commute": "Belastningsfaktor buss pendling",
+    "load_factor_car_average": "Belastningsfaktor bil genomsnitt",
+    "load_factor_car_commute": "Belastningsfaktor bil pendling",
+    "load_factor_increase_heavy_trucks": "Belastningsfaktorökning (tunga lastbilar)",
+    "load_factor_increase_light_trucks": "Belastningsfaktorökning (lätta lastbilar)",
+    "machinery_efficiency_improvement": "Maskineri effektivitetsförbättring",
+    "meeting_length": "Möteslängd",
+    "mileage_external_meeting": "Avstånd externa möten",
+    "population": "Befolkning",
+    "population_growth": "befolkningstillväxt",
+    "proportion_business_travellers": "Andel affärsresande",
+    "proportion_commutes": "Andel pendling",
+    "proportion_office_workers": "Andel kontorsarbetare",
+    "proportion_working": "Andel arbetare",
+    "retrofitting_savings_factor_commercial_buildings": "Efteranpassningsbesparningsfaktor (kommersiella byggnade)",
+    "retrofitting_savings_factor_housing": "Efteranpassningsbesparningsfaktor (hus)",
+    "rooftop_area_pv_start": "Takareal PV start",
+    "savings_from_route_optimisation": "Besparningar från ruttoptimering",
+    "stock_direct_electricity_commercial": "Stock direktverkande elektricitet (kommersiell)",
```

```suggestion
    "stock_direct_electricity_commercial": "Stock direktverkande elektricitet (kommersiella byggnader)",
```
Ordet "stock" fungerar inte bra på svenska, men jag lämnar det därhän tills vi pratat med Tomer och Gunnar om ett lämpligt ord på svenska.